### ISSUE TITLE
the performance on outdoor data is not so good?

### ISSUE THREAD

**Issue by Neo-96** [rpautrat/SuperPoint#219]
Hi, I tried the network with the pretrained weights on a few pairs of images, and I found that the performance is better when the input is indoor data or some scenes like indoor environment. When input outdoor image with sky and cloud as its background, there are more keypoints on the region of sky and cloud than the building region and other regions, and these are not good features for this scene. Is this because the pretrained weights is more suitable for indoor environments? Is there any outdoor model can solve the problem?

**Comment by rpautrat** [comment]
Hi,
The released model has been trained on the COCO dataset, which is more an objects dataset and is thus not very well suited for visual localization. I would recommend fine-tuning it on an outdoor visual localization dataset, such as MegaDepth. Ideally, if you have ground truth masks for the sky part of the images, you could even train the network to ignore the sky areas to avoid the issue you mentioned.

**Comment by Neo-96** [comment]
Thanks for your reply!

**Comment by rpautrat** [comment]
I am closing the issue now, feel free to reopen it if you have more questions.
